README.md

```diff
@@ -29,28 +29,130 @@ Anywhere `markdown` is allowed in a yaml, the following syntax is allowed:
+    attacks?: CreatureAttack[] = []
+    actions?: CreatureAction[] = []
+    reactions?: CreatureReaction[] = []
+    legendary_actions?: CreatureAction[]
+    legendary_action_points?: uint | string = legendary_actions === undefined ? undefined : “1 legendary action point per player”
+}
+
+// where:
+
+type CreatureSize = "Minuscule" | "Tiny" | "Small" | "Medium" | "Large" | "Huge" | "Gargantuan"
+
+type CreatureType = "Aberration" | "Beast" | "Celestial" | "Construct" | "Dragon" | "Elemental" | "Fey" | "Fiend" | "Giant" | "Humanoid" | "Monstrosity" | "Ooze" | "Plant" | "Undead"
+
+type CreatureTrait = {
+    name: string
+    description: contextual_markdown
+}
+
+type CreatureAttack = {
+    name: string
+    proximity: "melee" | "ranged"
+    medium: "weapon" | "spell"
+    properties?: string[] = [] // e.g. ["finesse", "thrown (20/60 ft.)"]
+    hit: contextual_markdown
+}
+
+type CreatureAction = {
+    name: string
+    cost: int
+    description: contextual_markdown
+}
+
+type CreatureReaction = {
+    name: string
+    cost: uint // 0 => free reaction, 1 => normal reaction
+    trigger: contextual_markdown
+    description: contextual_markdown
+}
+```
+
+In the above, `contextual_markdown` supports everything that `markdown` supports in addition to the special "@" syntax below, which causes the item to be rendered differently depending on context. In this case, "context" means either as a typical DM-controlled creature, a creature summoned by a spell, or a druid's Wild Shape form (the latter two contexts are currently identical).
+- `@save({ability_score}, {ability_score})`
+    - first arg: kind of save the target makes
+    - second arg: stat used to set the DC
+    - renders as either e.g. “DC 14 Constitution saving throw” or “Constitution saving throw against your spell save DC”, depending on context
+    - example usage: “the target makes a @save(dex, cha) or takes 8d6 fire damage”
+- `@check({ability_score | skill}, {ability_score})`
+    - same as `@save`, but renders as either e.g. "DC 10 Athletics check" or "Athletics check against your spell save DC", depending on context
+- `@plus({ability_score})`
+    - renders as either e.g. “ - 1” or “ + your spellcasting ability score”, depending on context
+    - example usage: “1d8@plus(str) slashing damage”
+- in all of the above, the following values are allowed:
+    - ability_score: str, dex, con, int, wis, cha
+    - skill: acrobatics, animal_handling, arcana, athletics, deception, history, insight, intimidation, investigation, medicine, nature, perception, performance, persuasion, religion, sleight_of_hand, stealth
+
+## Miscellaneous Types
+
+```ts
+type Skill = "Acrobatics" | "Animal Handling" | "Arcana" | "Athletics" | "Deception" | "History" | "Insight" | "Intimidation" | "Investigation" | "Medicine" | "Nature" | "Perception" | "Performance" | "Persuasion" | "Religion" | "Sleight of Hand" | "Stealth"
+
+type AbilityAcronym = "str" | "dex" | "con" | "int" | "wis" | "cha"
```

Comment:
This is definitely a nit, but those are just abbreviations, not acronyms.

Reply:
oh that is no mere nit, that is a fatal error that i have to fix